_episodes/06-script.md

```diff
@@ -511,6 +511,17 @@ Of course, this introduces another tradeoff between flexibility and complexity.
 > > wc -l $1/*.$2 | sort -n | tail -n 2 | head -n 1
 > > ```
 > > {: .source}
+> >
+> > The first part of the pipeline, `wc -l $1/*.$2 | sort -n`, counts
+> > the lines in each file and sorts them numerically (largest last).
+> > Remember, though,that when there's more than one file, `wc` outputs a
```

Comment:
~~Remember, though,that w~~When there's more than one file, `wc` **also** outputs a